src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandArgumentsProviderTests.cs

```diff
@@ -58,15 +58,14 @@ public void LocalDebuggerCommandArgumentsProvider_NullAsConfiguredProject_Throws
         }
 
         [Theory]
-        [InlineData("consoleapp1.exe", "consoleapp1", false, "arg1 arg2")]
-        [InlineData("consoleapp1.exe", "consoleapp1", true, "arg1 arg2")]
-        public async Task LocalDebuggerCommandArgumentsProvider_EmptyCommand_ReturnsExec(string executable, string dll, bool useOutputGroups, string args)
+        [InlineData(@"C:\Temp\consoleapp1.exe", "arg1 arg2", false,  @"exec C:\Temp\consoleapp1.dll arg1 arg2")]
```

Comment:
Missing the quotes around the binary path. Also, can you remove the unused System.IO using at the top?